src/components/CourseSetEdit.vue

```diff
@@ -12,70 +12,83 @@
         @click="switchCurrentCourseSet(courseSet)"
         >{{ courseSet }}</b-dropdown-item
       >
+      <div class="dropdown-divider"></div>
       <b-dropdown-item v-b-modal.courseSet-modal>
         <font-awesome-icon
           title="Settings"
-          :icon="['fas', 'plus']"
-        ></font-awesome-icon>
-        Add /
-        <font-awesome-icon
-          title="Settings"
-          :icon="['fas', 'trash']"
-        ></font-awesome-icon>
-        Remove</b-dropdown-item
-      >
+          :icon="['fas', 'pen']"
+        ></font-awesome-icon
+        ><!-- this color is kind of ugly for an icon.  perhaps a dark gray instead? -->
+        Edit
+      </b-dropdown-item>
     </b-nav-item-dropdown>
 
     <b-modal id="courseSet-modal" title="Course Set Settings">
       <p>
         Course sets allow you to save and switch among multiple selections of
         courses/sections.
       </p>
-      <b-input-group>
-        <b-form-input
-          v-model="newCourseSetName"
-          :state="newCourseSetExists"
-          placeholder="Course Set Name"
-          aria-lable="Course Set Name"
-          trim
-          @keyup.enter="createNewCourseSet"
-        ></b-form-input>
-        <b-input-group-append>
-          <b-button
-            @click="createNewCourseSet"
-            style="
-              border-top-right-radius: 0.25rem;
-              border-bottom-right-radius: 0.25rem;
-            "
-            :disabled="!newCourseSetExists"
-            :title="newCourseSetExists ? '' : 'Disabled'"
-            >Add Course Set</b-button
-          ></b-input-group-append
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item">
+          <h5 class="mb-0">Course Sets:</h5>
+        </li>
+        <li
```

Comment:
These should be divs or have their styling changed so the table's row lines aren't visible and the spacing between consecutive course sets is significantly reduced.

Reply:
Why?  I deliberately switched to a table format with lines to give a little bit of separation between each element.  I could potentially inset the separator between each of the course sets slightly to make it clear that the course sets are one group.  I could also remove the left padding on the title and input field for a similar effect.

Reply:
In my opinion they're too far apart.  It's fine if you want a bit of spacing between them, but it shouldn't be this pronounced.